sonic_api.yaml

```diff
@@ -1255,6 +1255,285 @@ paths:
           schema:
             $ref: '#/definitions/Error'
 #----------------------------------------------
+# QinQ subinterface object
+#----------------------------------------------
+  '/config/subinterface/qinq/{if_name}/{outer_tag}/{inner_tag}':
+    put:
+      summary: Create/update QinQ router interface for a physical interface
+      description: If the interface is already created, only QinQ attributes can be updated.
```

Comment:
Please mention what attributes can be modifed

Reply:
I don't see anything which cannot be updated in QinQEntry object. Exceptions can be added in implementation for lets say if some field updation is dependent on some other resources.